apps/condo/index.js

```diff
@@ -143,6 +143,53 @@ const apps = () => {
 
 /** @type {(app: import('express').Application) => void} */
 const extendExpressApp = (app) => {
+    const runtimeStats = {
+        activeRequestsIds: new Set(),
+        activeRequestsCountByType: {
+            graphql: 0,
+            oidc: 0,
+            other: 0,
+        },
+    }
+
+    app.use(function runtimeStatsMiddleware (req, res, next) {
+        const url = req.url
+        const method = (req.method || 'get').toLowerCase()
+
+        const isPost = method === 'post'
+
+        const isAdminApi = url.startsWith('/admin/api')
```

Comment:
```suggestion
        const isGraphql = url.startsWith('/admin/api')
```

Reply:
Should we also parse exact path here? Like:
```
const url = new URL(req.url)

 const isGraphql = url.pathname === '/admin/api' 
```

Reply:
I think no. What is the profit?

Reply:
Because some weird request can affect your stat

Like request to `/admin/api/something/resolving/404`

Reply:
And since your target is to separately monitor amount of GQL requests, you want to separate specific endpoint, so other routes will count not as gql, but as others instead

Reply:
> 

`isAdminApi` is about the URL itself.